src/dcat-us/compile-dcat-feed.ts

```diff
@@ -25,6 +22,32 @@ function removeUninterpolatedDistributions(distributions: any[]) {
   return distributions.filter((distro) => !(typeof distro === 'string' && distro.match(/{{.+}}/)?.length));
 }
 
+function generateDcatItem(feedTemplate, feedTemplateTransforms, geojsonFeature) {
+  const defaultFields = {
+    '@type': 'dcat:Dataset'
+  };
+
+  // moving feature properties fields at the top level
+  // and geometry to second as the values for the properties
+  // can be easily referenced in template
```

Comment:
Comments here are not necessary.  They should only be used when the code cannot be written in away that explains itself. See https://bpoplauschi.github.io/2021/01/20/Clean-Code-Comments-by-Uncle-Bob-part-2.html.

Reply:
That is great point. I wanted to mention why we are restructuring  geojson properties and geometry like I am doing right now as opposed to passing the whole object `geojsonFeature`.  This would relate to how we are mapping the values in the template. But, I have removed the comment.